figures/vis/panel-internal-class.mmd

```diff
@@ -39,8 +48,9 @@ classDiagram
     PanelPlugin "1" *-- "1" CorePanel : Provides
     CorePanel "1" *-- "0..1" GlobalView : Renders
     CorePanel "1" *-- "0..1" RegionView : Renders
-    GlobalView "1" *-- "1" ForceGraph : Renders
-    RegionView "1" *-- "1" ForceGraph : Renders
+    GlobalView "1" *-- "1" TreeDisplay : Renders
+    RegionView "1" *-- "1" TreeDisplay : Renders
```

Comment:
`TreeDisplay` does not currently work like this. It does not take the same props as `ForceGraph`.